frontend/src/components/auth/LoginForm.vue

```diff
@@ -101,23 +109,55 @@
             }),
             onLogin() {
                 if (this.$refs.form.validate()) {
+                    this.isLoading = true;
                     this.login({
                         email: this.email,
                         password: this.password
                     }).then(() => {
+                        this.isLoading = false;
                         this.$emit("success");
                         this.showSuccessMessage('Logged in');
                     }).catch((error) => {
+                        this.isLoading = false;
                         this.showErrorMessage(error);
                     });
                 }
             }
+        },
+        computed: {
+            signInText() {
+                return this.isLoading?'SIGN INNING...':'SIGN IN';
```

Comment:
not sure that such word like "SIGN INNING" exists, better replace it to "Processing..." or something like that